packages/sdks/src/scripts/init-editing.ts

```diff
@@ -50,6 +50,7 @@ export const setupBrowserForEditing = (
           // scope our '+ add block' button styling
           supportsAddBlockScoping: true,
           supportsCustomBreakpoints: true,
+          supportsXSmallBreakpoint: true,
```

Comment:
Since this breakpoint isn't implemented for React Native, we should move this behind a check:

```suggestion
          supportsXSmallBreakpoint: TARGET === 'reactNative' ? false : true,
```